packages/p2p-v2/jest.config.js

```diff
@@ -3,7 +3,9 @@ const baseConfigForPackages = require('../../jest.config.base');
 module.exports = {
     ...baseConfigForPackages,
     moduleNameMapper: {
+        '\\.css$': '<rootDir>/../../__mocks__/styleMock.js',
         '\\.s(c|a)ss$': '<rootDir>/../../__mocks__/styleMock.js',
         '^.+\\.svg$': '<rootDir>/../../__mocks__/fileMock.js',
     },
+    transformIgnorePatterns: ['/node_modules/(?!@deriv-com/ui).+\\.js$'],
```

Comment:
This tells Jest to ignore `@deriv-com/ui` since the files are in ESM and Jest doesn't know how to handle ESM at the moment.
We also need to add mocking for styles `css` files